skill-tree /src/main/java/com/RDS/skilltree/Endorsement/EndorsementResponse.java

```diff
@@ -0,0 +1,12 @@
+package com.RDS.skilltree.Endorsement;
+
+import lombok.*;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class EndorsementResponse<T> {
```

Comment:
can we have a generic class for all entities, this way we will end up creating the class for each response.

Reply:
Moved it into a common package so that can be used across all packages